### PR TITLE
added marketplace values to schemas

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10623

## Description
Added the option to specify a marketplace to the following playbook fields: inputs, scriptarguments, message, form, left, right, tasks.
Made it so the top hierarchy always wins (and is overridden) when collecting schema marketplace specific arguments.

### CODE REVIEW

**Comment by DinaMeylakh** [comment]
Tested manually: all keys that were changed in the schema, if available a subfield was also tested.
The test included: changing the yml to have a field:xsoar: value, running format+validate on it, running upload and seeing it change in an xsoar instance. 
playbookInputQuery was not tested as there was no working example in the content repo.

File: demisto_sdk/commands/common/schemas/playbook.yml [hunk 1]
```diff
@@ -400,6 +470,26 @@ schema;input_schema:
       type: any
 
 # playbook->inputSections
```
Comment (dantavori) [inline demisto_sdk/commands/common/schemas/playbook.yml:472]:
```suggestion
```

File: .changelog/4375.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: "Added the option to specify a marketplace to the following playbook fields: inputs, scriptarguments, message, form, left, right, tasks."
```
Comment (dantavori) [inline .changelog/4375.yml:2]:
```suggestion
- description: |-
  Added the option to specify a marketplace for certain playbook fields:
  * All input fields - *key*, *value*, *description*, *required*, *playbookInputKey*).
  * The following task fields: *scriptId*, *scriptarguments*, *loop.scriptArguments*, *message*, *form*.
  * The *left* and *right* options of *condition*/*builtincondition* task fields (and their nested fields).
```
This is very specific and should be documented somewhere for future reference.

File: demisto_sdk/commands/prepare_content/tests/marketplace_suffix_preparer_test.py [hunk 3]
```diff
@@ -15,6 +15,9 @@
     "some:xpanse": "xpanse",
     "some:xsoar_on_prem": "xsoar_on_prem",
     "some:xsoar_saas": "xsoar_saas",
+    "value": {"simple": "test value"},
+    "value:xsoar": {"simple": "test xsoar value"},
+    "value:marketplacev2": {"simple": "test marketplacev2 value"},
```
Comment (dantavori) [inline demisto_sdk/commands/prepare_content/tests/marketplace_suffix_preparer_test.py:20]: let's add some more of the new fields

**Review by dantavori — APPROVED** [review]
noice :) approving, but see some minor notes

**Pull request merged**